crates/telio-model/src/api_config.rs

```diff
@@ -247,6 +247,9 @@ pub struct Features {
     /// Flag to specify if keys should be validated
     #[serde(default)]
     pub validate_keys: FeatureValidateKeys,
+    /// Avoid sending periodic messages to peers with no traffic reported by wireguard
+    #[serde(default)]
+    pub skip_unresponsive_peers: bool,
```

Comment:
Damn, missed this yesterday. 

Can we move this `bool` into "FeatureDirect" as CPC is fully dependent on direct support. In a way it is nice, that this bool becomes "unsettable" if direct connections are disabled :).

Reply:
It is aleady moved in the branch with natlab tests: https://github.com/NordSecurity/libtelio/blob/LLT-4152-disable-cpc-and-session-keeper-natlab-test/crates/telio-model/src/api_config.rs#L181-L193 and we agreed just agreed that there is no need to backport it here. I will make it on by default on that branch also.